metatron-app/metatron-discovery
# #3721 Add color settings for measures in kpi charts

### Description
Added color setting for each measure in KPI

**Related Issue** : #3721 


### How Has This Been Tested?
1.  Create label chart
2. Color settings for measurements on the shelf
3. Check the dashboard

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
